src/main/java/io/github/ust/mico/kafkafaasconnector/kafka/MicoCloudEventImpl.java

```diff
@@ -70,6 +70,37 @@
         return this;
     }
 
+    /**
+     * Create a shallow copy of this cloud event.
+     *
+     * @return the copy
+     */
+    public MicoCloudEventImpl<T> copy() {
```

Comment:
You could realize this as via the copy constructors pattern or use [BeanUtils ](http://commons.apache.org/proper/commons-beanutils/javadocs/v1.9.3/apidocs/index.html) which provide copy functionality without the need to manually maintain it.